include/rocksdb/iostats_context.h

```diff
@@ -48,6 +68,8 @@ struct IOStatsContext {
   uint64_t cpu_write_nanos;
   // CPU time spent in read() and pread()
   uint64_t cpu_read_nanos;
+
+  FileIOByTemperature file_io_stats_by_temperature;
```

Comment:
What's the difference between the one here and the one in PerfContext?

Reply:
I guess the difference is bytes and count? Can we measure both in RandomAccessFileReader and put it here? In that way, we don't have to touch BlockBasedTableReader, so the logic would be simpler. Would that work?

Reply:
Sure, I will move the counter to iostatus and measure them in RandomAccesssFuleReader::Read()